solidity/contracts/helpers/TestTokenTimeWeightedAverage.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: SEE LICENSE IN LICENSE
+pragma solidity 0.6.12;
+
+import "../liquidity-protection/TokenTimeWeightedAverage.sol";
+
+contract TestTokenTimeWeightedAverage is TokenTimeWeightedAverage {
+    uint256 public currentTime = 1;
```

Comment:
We already have this infrastructure in about 3 or 4 different places; might wanna consider reusing it somehow.

Reply:
Perhaps. I'll add it to my todos

Reply:
Done